src/app/src/test/kotlin/org/imperial/mrc/hint/integration/adr/ADRTests.kt

```diff
@@ -197,7 +197,7 @@ class ADRTests : SecureIntegrationTests()
         return if (isAuthorized == IsAuthorized.TRUE)
         {
             testRestTemplate.postForEntity<String>("/adr/key", getPostEntityWithKey())
-            val resultWithResources = testRestTemplate.getForEntity<String>("/adr/datasets?showInaccessible=true")
+            val resultWithResources = testRestTemplate.getForEntity<String>("/adr/datasets?showInaccessible=false")
```

Comment:
We've been getting pretty consistent ADR test failures, so I was suspicious it was a dataset issue rather than dev server being intermittent. Not really sure why we were ever testing with inaccessible resources here, but this seemed to help when running tests locally, as failures were around trying to access unavailable resources.